pycryptobot.py

```diff
@@ -88,7 +88,8 @@ def calculateMargin(buy_size: float=0.0, buy_filled: int=0.0, buy_price: int=0.0
     sell_size = (sell_percent / 100) * ((sell_price / buy_price) * buy_size)
 
     if sell_fee == 0.0 and sell_taker_fee > 0.0:
-        sell_fee = sell_size * sell_taker_fee
+        #sell_fee = sell_size * sell_taker_fee
```

Comment:
I don't think this is correct. Fees are based on the trade size not the price. For example if the Bitcoin price is £40,000 and your trade is £1,000 the fees is calculated on the £1,000 not £40,000.

Reply:
yes you are perfectly right i understand what you mean

Reply:
What is this pull request actually changing now? It looks like you just have undone your previous pull request? Does this update work? Have you confirmed with Mario?

Reply:
This was before we talked, i'm checking it actually

Reply:
I'll make a pull as soon it will be ok

Reply:
reverted on my second commit, i'm waiting for a chack from Mario as well